server/simulation/missionProgress.js

```diff
@@ -5,9 +5,15 @@ module.exports = {
     status: 'travelling_pickup',
     nextVehicleStatus: 'landing_pickup',
     nextMissionUpdate: 'landing_pickup',
-    conditionForNextUpdate: mission => {
+    conditionForNextUpdate: (mission, vehicle) => {
       let elapsedTime = Date.now() - (parseFloat(mission.user_signed_at) + parseFloat(mission.time_to_pickup));
-      return elapsedTime > 0;
+      let isAtTerminal = false;
+
+      if(vehicle.coords.lat == mission.pickup_latitude && vehicle.coords.long == mission.pickup_longitude) {
```

Comment:
That's great.
However float number comparison with === is most likely to fail (e.g. 0.00000001 !== 0.00000002).
Can you use the previous method of using a Diff Delta?

Reply:
For sure! is `0.00000005` as the diff delta ok?

Reply:
I think 10e-4 would be enough :)

Reply:
And sorry for the delay - didn't notice your comment.

Reply:
If 1x10e-4 is .0001, isn't that still a large distance when using GPS coords?  If I use a deltaThreshold of 10e-8 then it works correctly.  I'm getting the delta amount like so for long and lat:
`Math.abs(vehicleLat - pickupLat)` and `Math.abs(vehicleLong - pickupLong)`
Not sure if maybe there is a better way of finding the distance between two GPS coords.

Reply:
Sounds good to me :)